test/sagemaker_tests/pytorch/training/integration/sagemaker/test_distributed_operations.py

```diff
@@ -236,10 +235,7 @@ def test_smmodelparallel_smdataparallel_mnist(instance_types, ecr_image, py_vers
     This test has been added for SM DataParallelism and ModelParallelism tests for re:invent.
     TODO: Consider reworking these tests after re:Invent releases are done
     """
-    _, image_framework_version = get_framework_and_version_from_tag(ecr_image)
-    image_cuda_version = get_cuda_version_from_tag(ecr_image)
-    if not (Version(image_framework_version) in SpecifierSet(">=1.6,<1.8")) or image_cuda_version != "cu110":
-        pytest.skip("Model Parallelism only supports CUDA 11 on PyTorch 1.6 and PyTorch 1.7")
+    validate_or_skip_smmodelparallel(ecr_image)
```

Comment:
We can't skip both smdataparallel and smmodelparallel tests just because modelparallel isn't compatible.

Maybe we could run just the smdataparallel script here if modelparallel doesn't work.

The purpose of this test is to function as a pre-release validation check in each region.

Reply:
I wasn't sure what the purpose of this test was exactly. Looks like the skip was previous behavior too. I can make the test partially execute only smdataparallel